client-app/ui-kit/components/molecules/button/vc-button.vue

```diff
@@ -62,7 +62,7 @@ export interface IEmits {
 }
 
 interface IProps {
-  color?: "primary" | "secondary" | "success" | "info" | "neutral" | "warning" | "danger";
+  color?: "primary" | "secondary" | "success" | "info" | "neutral" | "warning" | "danger" | "accent";
```

Comment:
`primary` and `secondary` is already primary and secondary accent colors. What `accept` mean here? Is it tertiary accent?

Reply:
https://www.figma.com/file/RAvri6to8nVBUDpDQCBvUA/Mercury-UI-kit?type=design&node-id=12936-20638&mode=design&t=01GZ1gehGyAO3ayO-4